connectors/connectors/day-trade/day-trade-get-connector/pom.xml

```diff
@@ -156,7 +156,17 @@
           </execution>
         </executions>
       </plugin>
-
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>default-jar</id>
```

Comment:
I don't understand: why are we no longer creating JARs for connectors? And if this is true for all connectors now, should we just put this in connectors parent, we'll surely create a connector without `maven-jar-plugin` disabled...

Reply:
We do generate a jar.  It's the camel-connector-maven-plugin:2.20.0:jar which creates the jar.  We need to disable the default maven-jar-plugin generation since that fails when it notices the jar has already been created.